plugins/chat/assets/javascripts/discourse/components/chat/thread/header.js

```diff
@@ -37,7 +38,7 @@ export default class ChatThreadHeader extends Component {
   }
 
   get label() {
-    return this.args.thread.escapedTitle;
+    return htmlSafe(this.args.thread.escapedTitle);
```

Comment:
Why the indirection here with `escapedTitle`? Couldn't we do something like this, and let Ember handle the escaping?

```
get label(){
  return thread.title;
}
```

Reply:
Hm yes indeed, I changed it :+1: